twpca/model.py

```diff
@@ -415,6 +425,24 @@ def predict(self, X=None):
 
             return pred
 
+    def dump(self):
+        """Serializes model variables"""
+        return self._sess.run(self._vars)
```

Comment:
`self._vars` are all of the tensorflow variables. The user shouldn't really worry about these, but they are useful to serialize and reinstantiate the model.

`self._params` (which is accessible via `self.params` which gives numpy arrays) holds things the user *does* care about. I.e. the low-dimensional factors and the warping functions.